db/models.py

```diff
@@ -14,3 +16,37 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
+
+    def __str__(self) -> CharField:
```

Comment:
The return type of the `__str__` method should be `str`, not `CharField`. Update the type hint to `-> str`.